client-templates/nexus2/.env.sample

```
@@ -27,3 +27,6 @@ BROKER_HEALTHCHECK_PATH=/healthcheck
 BROKER_CLIENT_VALIDATION_URL=$BASE_NEXUS_URL/nexus/service/local/status
 
 BROKER_CLIENT_VALIDATION_JSON_DISABLED=true
+
+# Disable X-Forwarded-For headers so Nexus doesn't return npm tarball uri pointing to the broker server
+removeXForwardedHeaders=true
```

Recommend making it consistent with the other env vars.
```suggestion
REMOVE_X_FORWARDED_HEADERS=true
```